### DISCUSSION TITLE
SIEGE INSIGHTS  Error: u.filter is not a function

### DISCUSSION THREAD

**Discussion by CedricLee2017** [Xzandro/sw-exporter#428] [Q&A]
Dear Everyone

I have been using for exporter for siege war for years. It works fine until last night..
When i uploaded the jason file, it shows a message "Error: u.filter is not a function" 

Could anyone advice me what to do?
Regards
Cedric

**Comment by Xzandro** [comment]
Probably better to contact the author of the plugin itself, since this is not an internal SWEX plugin.

**Reply by CedricLee2017** [reply]
thx for your reply. I tried to reach author but no replies so far... my friend send me his jason file and then i upload to siege insight website. it can work. So i think there is something wrong when i export the jason file.
